### PR TITLE
Add adobe tracking code

### PR DESCRIPTION
**Pull request by normanargueta**

### CODE REVIEW

File: views/index.ejs [hunk 1]
```diff
@@ -96,5 +100,80 @@
...
+          interactionType: '',
+          events: '',
+        },
+      };
+      (function(){
+        var width = window.screen.width;
+        var responsive = ( width < 768 ) ? "mobile" : ( width < 960 ) ? "tablet" : "desktop";
+        digitalData.user.profile.profileInfo.responsiveSiteVersion = responsive;
+      })();
+      if (typeof _satellite !== "undefined") _satellite.pageBottom();
```
Comment (jeffswartz) [inline views/index.ejs:175]: Why not `if (_statellite) _satellite.pageBottom()` (or `_satellite && _satellite.pageBottom()`)?

File: views/room.ejs [hunk 2]
```diff
@@ -400,6 +403,80 @@
...
+          interactionType: '',
+          events: '',
+        },
+      };
+      (function(){
+        var width = window.screen.width;
+        var responsive = ( width < 768 ) ? "mobile" : ( width < 960 ) ? "tablet" : "desktop";
+        digitalData.user.profile.profileInfo.responsiveSiteVersion = responsive;
+      })();
+      if (typeof _satellite !== "undefined") _satellite.pageBottom();
```
Comment (jeffswartz) [inline views/room.ejs:478]: Same comment as above: can we use `if (_statellite) _satellite.pageBottom()` (or `_satellite && _satellite.pageBottom()`)?

File: server/serverMethods.js [hunk 3]
```diff
@@ -35,6 +37,29 @@ function htmlEscape(str) {
...
+  }
+
+  return language;
+}
+
+function getUserCountry(req) {
+  const ip = req.headers['x-forwarded-for'] || req.connection.remoteAddress;
+  const geo = geoip.lookup(ip) || {};
+
+  return geo.country ? geo.country.toLowerCase() : '';
```
Comment (jeffswartz) [inline server/serverMethods.js:60]: Maybe use lodash: `return _.get(geo, 'country', '').toLowerCase;`?

File: views/room.ejs (hunk 2 above)
Comment (normanargueta) [reply views/room.ejs:478]: @jeffswartz I didn't want to change the code provided by the Adobe team:
Screen Shot 2020-04-13 at 13 47 00

What do you think? Should I change it?

File: server/serverMethods.js (hunk 3 above)
Comment (normanargueta) [reply server/serverMethods.js]: @jeffswartz  Changed, thanks for the feedback 👍

File: views/index.ejs [hunk 4]
```diff
@@ -28,6 +28,10 @@
     <script defer type="text/javascript" src="/js/vendor/lazy_loader.js"></script>
     <script defer type="text/javascript" src="/js/rtcApp.js"></script>
 
+    <% if (adobeTrackingUrl) { %>
```
Comment (rhainer) [inline views/index.ejs:31]: Looks like the updated requirements ask that we load the script with async. Should we add that in the code, as part of the adobeTracingUrl variable, or add a new variable?

Comment (normanargueta) [reply views/index.ejs:31]: @rhainer we should add that in the script tag another variable is not needed for that:
`<script src="<%=adobeTrackingUrl%>" async></script>`
check the `async` attribute

File: views/room.ejs (hunk 2 above)
Comment (jeffswartz) [reply views/room.ejs:490]: No, it's fine.

**Pull request merged**